pkgs/applications/graphics/sane/backends/brscan4/preload.c

```diff
@@ -0,0 +1,170 @@
+/* Brgen4 search for configuration under `/etc/opt/brother/scanner/brscan4`. This
```

Comment:
Thanks for the review. It's nice to know this generic way of doing things exists. I will changes
the file names to using dash when I get some free time.